### PR TITLE
Add links to vector database page

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Add links to vector database page

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _redesign_use_cases/search-vector-database.markdown [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: 'OpenSearch as a vector database supports a range of applications.
 feature_area_category_name: Search
 feature_area_solution_name: Vector Database
 how_to_get_started: 
-  - 'You can begin exploring OpenSearch&apos;s vector database functionality by downloading your preferred distribution. To learn more or start a discussion, join the Slack channel or check out our user forum and follow our blog for the latest on OpenSearch tools.'
+  - 'You can begin exploring OpenSearch&apos;s vector database functionality by exploring <a href="https://opensearch.org/docs/latest/search-plugins/vector-search/" target="_blank">Vector search</a> in our documentation. To learn more or start a discussion, join the Slack channel or check out our user forum and follow our blog for the latest on OpenSearch tools.'
```
Comment (natebower) [inline _redesign_use_cases/search-vector-database.markdown:8]:
```suggestion
  - 'You can get started with OpenSearch&apos;s vector database functionality by exploring <a href="https://opensearch.org/docs/latest/search-plugins/vector-search/" target="_blank">Vector search</a> in our documentation. To learn more or to start a discussion, join our public Slack channel, check out our user forum, and follow our blog for the latest on OpenSearch tools and features.'
```

Comment (natebower) [inline _redesign_use_cases/search-vector-database.markdown:8]: The first sentence would read better as "by exploring our vector search documentation." Also, let's add links to the Slack channel, the user forum, and the blog. Thanks!

**Review by natebower — COMMENTED** [review]
@kolchfa-aws Please see my comment/changes and let me know if you have any questions. Thanks!

**Comment by kolchfa-aws** [comment]
@krisfreedain @nateynateynate Could you review this PR?

**Review by nateynateynate — APPROVED** [review]
Looks good.

**Pull request merged**